web/angular-wallet/src/app/setup/account/account.service.ts

```diff
@@ -200,7 +200,7 @@ export class AccountService {
     return new Promise((resolve, reject) => {
       this.storeService.selectAccount(account)
         .then(account => {
-          // this.synchronizeAccount(account);
+          this.synchronizeAccount(account);
```

Comment:
This updates the recent transactions when account switches.

Reply:
why was it commented then?

Reply:
an old bug that no longer exists